lib/themes/dosomething/paraneue_dosomething/js/app.js

```diff
@@ -78,11 +79,12 @@ $(document).ready(function() {
   ShowMore.init();
 
   // Intialize takeover modal.
-  Takeover.init();
+  // Takeover.init();
 
   // Add fallback for HTML5 video on mobile tiles
   Tile.init();
 
+  let donation = new OrganDonation($('.takeover-container'));
```

Comment:
@DFurnes not sure i understand how the init function would work with classes...it seems we should check if the element we are looking for is available before instantiating the class like...

``` 
if ($('.takeover-container')) {
let donation = new OrganDonation($('.takeover-container'));
}
```

Or are you saying the init function gets called in the constructor of these classes, so we still instantiate the class on every page, but none of the code runs.